2021-02-06 06:30:11,980 INFO    MainThread:7752 [wandb_setup.py:_flush():70] setting env: {}
2021-02-06 06:30:11,980 INFO    MainThread:7752 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-06 06:30:11,981 INFO    MainThread:7752 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-06 06:30:11,981 INFO    MainThread:7752 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-06 06:30:11,981 INFO    MainThread:7752 [wandb_init.py:_log_setup():318] Logging user logs to /home/iiitd/Desktop/Ahmad/doa1/wandb/run-20210206_063011-18a9z9y8/logs/debug.log
2021-02-06 06:30:11,981 INFO    MainThread:7752 [wandb_init.py:_log_setup():319] Logging internal logs to /home/iiitd/Desktop/Ahmad/doa1/wandb/run-20210206_063011-18a9z9y8/logs/debug-internal.log
2021-02-06 06:30:14,696 INFO    MainThread:7752 [wandb_run.py:_console_start():1308] atexit reg
2021-02-06 06:30:14,696 INFO    MainThread:7752 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-02-06 06:30:14,697 INFO    MainThread:7752 [wandb_run.py:_redirect():1178] Redirecting console.
2021-02-06 06:30:14,697 INFO    MainThread:7752 [redirect.py:install():205] install start
2021-02-06 06:30:14,698 INFO    MainThread:7752 [redirect.py:install():220] install stop
2021-02-06 06:30:14,698 INFO    MainThread:7752 [redirect.py:install():205] install start
2021-02-06 06:30:14,700 INFO    MainThread:7752 [redirect.py:install():220] install stop
2021-02-06 06:30:14,700 INFO    MainThread:7752 [wandb_run.py:_redirect():1222] Redirects installed.
2021-02-06 06:32:40,460 INFO    MainThread:7752 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-02-06 06:32:40,461 INFO    MainThread:7752 [wandb_run.py:_restore():1250] restore
2021-02-06 06:32:40,461 INFO    MainThread:7752 [redirect.py:uninstall():224] uninstall start
2021-02-06 06:32:40,461 INFO    MainThread:7752 [redirect.py:_stop():279] _stop: stdout
2021-02-06 06:32:40,461 INFO    stdout    :7752 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-02-06 06:32:40,461 INFO    stdout    :7752 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-02-06 06:32:40,462 INFO    MainThread:7752 [redirect.py:_stop():285] _stop closed: stdout
2021-02-06 06:32:40,462 INFO    MainThread:7752 [redirect.py:_stop():291] _stop joined: stdout
2021-02-06 06:32:40,462 INFO    MainThread:7752 [redirect.py:_stop():293] _stop rd closed: stdout
2021-02-06 06:32:40,462 INFO    MainThread:7752 [redirect.py:uninstall():228] uninstall done
2021-02-06 06:32:40,462 INFO    MainThread:7752 [redirect.py:uninstall():224] uninstall start
2021-02-06 06:32:40,462 INFO    MainThread:7752 [redirect.py:_stop():279] _stop: stderr
2021-02-06 06:32:40,463 INFO    stderr    :7752 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-02-06 06:32:40,463 INFO    stderr    :7752 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-02-06 06:32:40,463 INFO    MainThread:7752 [redirect.py:_stop():285] _stop closed: stderr
2021-02-06 06:32:40,463 INFO    MainThread:7752 [redirect.py:_stop():291] _stop joined: stderr
2021-02-06 06:32:40,463 INFO    MainThread:7752 [redirect.py:_stop():293] _stop rd closed: stderr
2021-02-06 06:32:40,463 INFO    MainThread:7752 [redirect.py:uninstall():228] uninstall done
2021-02-06 06:32:40,809 INFO    MainThread:7752 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "8b539da3c58b4c54ac3c83dc8778ab1a"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 511
      total_bytes: 511
    }
  }
}

2021-02-06 06:32:42,812 INFO    MainThread:7752 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "f13e230f63e449b8a218855692107d31"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6599
      total_bytes: 6599
    }
  }
}

2021-02-06 06:32:44,054 INFO    MainThread:7752 [wandb_run.py:_show_files():1614] logging synced files
2021-02-06 06:32:44,056 INFO    MainThread:7752 [internal.py:handle_exit():139] Internal process exited
